external/fv3fit/fv3fit/keras/_models/shared/callbacks.py

```diff
@@ -34,6 +42,10 @@ def callback(self, epoch_result: EpochResult):
             self.train_loss_all.append(batch_log.loss)
             self.val_loss_all.append(batch_log.val_loss)
 
+        _log_to_wandb_if_initialized(
```

Comment:
> .fit_loop should be modified to take a list of tf.keras.callbacks.Callback objects if it doesn't already.

Yes, that was the issue with using the wandb keras callback in this PR- currently `.fit_loop` does not take a list of keras callbacks but instead wants a list of `Callable[[EpochResult], None]` which is a remnant of how it used to call `.fit` every epoch. I talked to Jeremy yesterday and decided that I was going to change `fit_loop` to just take in keras callbacks and remove the `EpochCallback` class. It seems out of the scope of this PR though. In that future PR, this class will be removed. Is it ok to leave this for now since it will be removed but will allow me to run sweeps in the meantime?

Reply:
Or, I could put this PR on hold until the one remove `EpochCallback` is merged.

Reply:
No problem if you are planning on submitting another pr soon. it's not hard to undo these changes.